w3a2.c

```diff
@@ -0,0 +1,40 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+    main()
+     { 
+	  int arr[10],a,p,i,j,temp;
+  
+       printf("enter array size --> ");
+       scanf("%d",&a);
+       
+       printf("enter array elements --> ");
+      
+	  for(i=0;i<a;i++) 
+	   {
+	      scanf("%d",&arr[i]);
+	   }
+	   
+       printf("enter kth smallest element to be searched -->\n");
+       scanf("%d",&p);
+		   
+      for(i=1;i<a;i++)

```

Comment:
what is the time complexity of the solution? 
Why do we need to sort the array? 
If we do need to sort the array can you think of a better sorting algorithm? 
What if the array size is huge - (1000000000000000000000) and you have to find 100000000 th smallest element?